service/src/main/resources/mail.properties

```diff
@@ -0,0 +1,6 @@
+application.mail.weburl=http://localhost:8081/users
```

Comment:
this property file needs to be in api to be able change them fast and easy.
On server our service is one of jars that located in libs folder, so to change values there i must unzip jar, change file and then zip it again.